src/PixiEditor/Models/Rendering/MemberPreviewUpdater.cs

```diff
@@ -70,6 +80,21 @@ private void RenderWholeCanvasPreview()
         doc.PreviewPainter.ProcessingColorSpace = internals.Tracker.Document.ProcessingColorSpace;
         doc.PreviewPainter.FrameTime = doc.AnimationHandler.ActiveFrameTime;
         doc.PreviewPainter.Repaint();
+
+        if(!renderMiniPreviews)
+            return;
+
+        if (doc.MiniPreviewPainter == null)
```

Comment:
This (and above) can be simplified using ??=

And maybe it would make sense to call the constructor in a factor method that takes in doc and internals so there's less repetitive code